hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -177,10 +177,7 @@ private void checkServersAndTables(Set<Address> servers, Set<TableName> tables,
               + " does not exist.");
     }
     RSGroupInfo srcGrp = new RSGroupInfo(tmpSrcGrp);
-    if (srcGrp.getName().equals(targetGroupName)) {
-      throw new ConstraintException("Target RSGroup " + targetGroupName +
```

Comment:
If a table have 10 regions, and only 1 region is on the wrong rsgroup. The move will only move the wrong region and don't move the other 9 regions?

Reply:
Yes. It will judge which regions should be moved. Regions in correct group will not be moved in the origin logic.